src/app/sessions/components/sessions-filter/sessions-filter.component.ts

```diff
@@ -71,5 +68,6 @@ export class SessionsFilterComponent implements OnInit {
 
   sendFilter() {
     this.filter.emit(this.filters);
+    this.sessionFilter$.next(this.filters);
```

Comment:
What is the difference between this line and one above, why can't regular filter emitter be used, or changed to this new static-typed one?